src/wordpress/backup.py

```diff
@@ -90,7 +89,7 @@ def get_daily_list(self):
         # build regex for filenames with found dates
         matches = "|".join([valid_date.strftime("%Y%m%d")
                             for valid_date in valid_dates])
-        file_regex = re.compile("({})\d+.list".format(matches))
+        file_regex = re.compile("{}\d+.list".format(matches))
```

Comment:
Tu dois encore remettre les () autour car sinon le `|` (ou) que tu fais dans la regex n'a pas de "début" ou de "fin".
```suggestion
        file_regex = re.compile("({})\d+.list".format(matches))
```

Reply:
Oui, je les avais enlever car les tests ne passe pas avec... je dois comprendre pourquoi encore